content/blog/2021-01-21-unpack-cfr.Rmd

```diff
@@ -0,0 +1,249 @@
+by the date on which their infection was detected.
+By contrast, reported case fatality rates are typically not cohort-based---the patients whose deaths are reported in the numerator are not in general the same patients whose confirmed infections show up in the denominator.
+Because case confirmation tends to precede reported deaths, these signals tend to be misaligned and are subject to fluctuation, even if the actual case fatality rate were fixed (so long as incidence does change). 
+Line-level data enable us to circumvent this problem. Moreover, the availability of age and gender data allows us both to track demographic shifts over time, and to perform age-stratified analyses of fatality rates.
+
+# Methods
+
+For details on the datasets and pre-processing, see [the full manuscript](https://arxiv.org/abs/2012.04825). To reduce noise in our time series plots, we compute 7-day lagged averages for all remaining plots in the post unless otherwise specified.
+
+## Separating Artifacts from True Improvements
+We argue that three main phenomena fuel a dramatic “artificial" decrease in CFR: increased testing capacity (H1), shifting age distributions (H2), and delays between detection and fatality (H3).To separate the effects of these artifacts from those of true improvements, we do the following:
+
+* To avoid artifacts from increased testing (H1), when considering treatment improvements, we examine changes in _hospitalization fatality rates_ rather than case fatality rates.
+
+* To establish and account for shifting age distributions (H2), we examine cases, hospitalizations, and deaths stratified by age groups: 0-9, 10-19, 20-29, 30-39, 40-49, 50-59, 60-69, 70-79, and 80+.
+
+* To account for delays between detection and fatality (H3), we take advantage of line-level data for each individual. For each date, we extract the cohort of individuals confirmed positive on that date, as well as whether those individuals eventually died or were hospitalized. To provide enough time for patients’ eventual hospitalizations or deaths to be recorded, we filter out rows with positive specimen dates within 30 days of the last time the data was updated. Since CDC data was last updated on December 31th, we only use data from December 1st or earlier. Florida data is updated daily, but we use the same time range as the CDC data in order to make the plots comparable.
+
+Taking the above three adjustments into account, our primary quantity of interest for treatment improvements is the age-stratified HFR. For the rest of the post, 
+we define CFR and HFR at day $t$ as follows:
+\begin{align*}
+    \text{CFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually die}}{\text{cases confirmed (or reported) at day $t$}}\\
+    \text{HFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized and die}}{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized}}
+\end{align*}
```

Comment:
We use KaTeX to render math in HTML, so this needs to be
```suggestion
$$
\begin{aligned}
    \text{CFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually die}}{\text{cases confirmed (or reported) at day $t$}}\\
    \text{HFR}_t &= \frac{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized and die}}{\text{cases confirmed (or reported) at day $t$ that eventually get hospitalized}}
\end{aligned}
$$
```

Reply:
Done.